### PR TITLE
Bug fixing

### LINKED ISSUE #3670: [Bug] `economy/ycrv` is broken and has issues giving data from other countries
No data is given for `ycrv`, see:
image

Also, we list that it is possible to provide other countries which is false. This is due to the fact functionality from Investing.com no longer works and that is still baked into the command.

image

### LINKED ISSUE #3674: [IMPROVE] fred search should allow multiple words
In general, when searching for a term we should allow spaces

### PR DESCRIPTION
**Pull request by jmaslek**
Fixes #3670 (Also changes the sdk function slightly)

Fixes #3674

### CODE REVIEW

**Comment by montezdesousa** [comment]
I'm getting this warning when inputting a date that is not available, in this case `2022-12-01` onwards. In this case we should fallback to the last available as before imo

Screenshot 2022-12-02 at 09 34 33

**Comment by montezdesousa** [comment]
> @montezdesousa I have the following. Without a date argument, it passes the last date data is available: Screenshot 2022-12-02 at 9 51 43 AM
> 
> If I enter a weekend, it gets the next day: Screenshot 2022-12-02 at 9 52 59 AM
> 
> And if I enter the future, new message:
> 
> ```
> 2022 Dec 02, 09:52 (🦋) /economy/ $ ycrv -d 2040-10-10
> 
> Date cannot be today or in the future
> Yield data not found for 2040-10-10.
> ```

oof, sorry I must be looking to wrong branch, this ok in yours

**Comment by montezdesousa** [comment]
Seems to work fine for me
Screenshot 2022-12-02 at 17 41 32

**Review by montezdesousa — APPROVED** [review]

File: openbb_terminal/economy/economy_controller.py [hunk 1]
```diff
@@ -675,9 +675,9 @@ def call_fred(self, other_args: List[str]):
         )
         if ns_parser:
             parameters = list_from_str(ns_parser.parameter.upper())
-
+            print(ns_parser.query)
```
[the remark this answers is not in the document]
Comment (jmaslek) [reply openbb_terminal/economy/economy_controller.py:678]: rip

**Pull request merged**